bindings/java/src/org/sleuthkit/datamodel/OsAccountRealm.java

```diff
@@ -143,4 +221,52 @@ public static RealmNameType fromID(int typeId) {
 		}
 	}
 	
+	/**
+	 * Set the realmName if it is not already set.
+	 * 
+	 * @param name Realm name to set.
+	 */
+	public void setRealmName(String name) {
+		if (Objects.isNull(this.realmName) && Objects.nonNull(name)) {
+			this.realmName = name;
```

Comment:
call updateSignature

Reply:
I intentionally didn't update signature here.

If we are updating the name -  it mean name was earlier null, which means we knew the addr and hence  signature was set to addr.

Do we want to go from addr to name, knowing that addr is more definitive?